Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableSystemComponent.cpp

```diff
@@ -20,11 +21,27 @@ namespace AzFramework
     {
         Spawnable::Reflect(context);
         SpawnableMetaData::Reflect(context);
+        EntitySpawnTicket::Reflect(context);
 
         if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context); serializeContext != nullptr)
         {
             serializeContext->Class<SpawnableSystemComponent, AZ::Component>();
             serializeContext->RegisterGenericType<AZ::Data::Asset<Spawnable>>();
+
+            if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+            {
+                editContext->Class<EntitySpawnTicket>("EntitySpawnTicket", "An object used to spawn prefabs with SpawnableEntitiesInterface.");
+            }
+        }
+
+        if (auto behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
+        {
+            behaviorContext->Class<EntitySpawnTicket>("EntitySpawnTicket")
```

Comment:
question: Why does this have to be reflected from both SpawnableEntitiesInterface and SpawnableSystemComponent? Isn't reflecting it from one class enough?

Reply:
this was typo, thanks for catching I was actually trying to figure out where duplicate registration error was coming from!